### DISCUSSION TITLE
Timeline - how to set the range to more than default current day

### DISCUSSION THREAD

**Discussion by mattpgreen** [TerriaJS/terriajs#7245] [Q&A]
Hi - this must be a simple thing that I just haven't worked out.

How can a user set the range of time shown by the Timeline along the bottom of the window?  I only seem to be able to see the 24hrs around the CurrentTime.  This doesn't help fully when looking for shading events across the full year of seasons.  Also, ideally a user could zoom in so the slider covered only say 1hr so that fine detail in event movement say be seen.
This also seems like it should be both a global setting and an item by item setting - global so that the interaction between multiple displayed items can be checked - this is the use case where a proposed development is placed into an existing environment and shading both ways is to be assessed. - Thanks, Matt

**Comment by na9da** [comment]
Hi @mattpgreen, 

It is currently not possible for a user to set the range of the timeline. I have linked your request to this related issue - https://github.com/TerriaJS/terriajs/issues/4199
